assets/lambda/hai_lambda_glue_trigger.py

```diff
@@ -2,9 +2,9 @@
 
 import json
 import os
-import urllib.parse
 
 import boto3
+from capepy.aws.lambda_ import BucketNotificationRecord
 
 glue_client = boto3.client("glue")
```

Comment:
In a perfect world. The current capepy isn't as fully featured as it should be. This can either be something we do some planning and architecting before this is merged or we can do it after